benchmarks/clients/rumqttsync.rs

```diff
@@ -35,8 +35,8 @@ pub fn start(id: &str, payload_size: usize, count: usize) -> Result<(), Box<dyn
 
     let mut acks_count = 0;
     let start = Instant::now();
-    for event in connection.iter() {
-        if let Ok(Event::Incoming(Incoming::PubAck(_))) = event {
+    for event in connection.iter().flatten() {
```

Comment:
I had not known about `flatten()`, nice usage. Should we add a comment atop or will this be self-explanatory?

Reply:
I can add a comment.

Reply:
Actually I'll just revert this change, after thinking more about it I don't like it anymore.